### DISCUSSION TITLE
做菜翻车记--非日常料理篇 | /posts/fanche2/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#595] [Announcements]
如果一个人学做菜的过程中没翻过车，那是不可能的。要是真的有这样的人，那一定是神仙下凡，不然就是时空穿越者，事先知晓了一切。

https://yuanfan.rbind.io/posts/fanche2/

**Comment by residualsun1** [comment]
图二板栗的糖浆都泛光了……看着很诱人。湖北喜欢吃甜吗？感觉我们广东对甜没有抵抗力。

**Reply by yihui** [reply]
本湖北人路过，表示多数情况下都不喜欢甜。图 4 让我想起小时候去走亲戚时，进门主人就会给每个人端来一碗叫“鸡蛋茶”的东西，里面就是图 4 那样一个鸡蛋，然后开水里还有很多糖和猪油。其实那是高规格的待客方式，但那个甜得腻死人的甜法让我从小就很不喜欢甜食。长大了之后才被甜味驯化了很多。

**Reply by residualsun1** [reply]
拒绝猪油，好吃的东西沾上猪油都不好吃了。

之前和朋友去武汉玩，他们买过甜酒鸡蛋，里面只有醪糟和鸡蛋，这个就挺好吃的。

**Reply by earfanfan** [reply]
@residualsun1
其实猪油在我这90后记忆中是很好吃的。在记不大清的很多年以前，我们那会吃一种不知道是什么粉的鲜面粉子，吃法就是在碗里倒入鲜面粉子，然后一边倒入开水一边慢慢搅拌，最后要是能加上一点猪油拌进去，是非常香的。

湖北这边也有许多食物是蘸糖吃的，比如糍粑是蘸白糖或红糖的，豆腐脑是放白糖的。还有一样传说中的糖莲藕，我吃得少。

大概十年前我去过一次广州，印象中那边流行喝糖水还有喝汤，当然糖水是甜的，汤一般是咸的。

**Reply by residualsun1** [reply]
小时候我奶奶总是让我吃带白色肥猪油的猪肉，从那时候起我就很讨厌猪油。

蘸红糖的糍粑好吃。

我家有时候会用苹果、雪梨或者木瓜煮汤，咸中带有一部分甜味，但是是很香的。有时候猪骨可能煮出来把汤弄腻，但有水果喝起来就不会腻。或许你啥时候也可以试试（不知道这算不算广东的特色）

**Comment by yihui** [comment]
花生米看颜色那么浅应该是还没熟，再多炒五分钟试试。我从来不洗，要是非要洗的话，速速洗一下之后就下锅，先把水分炒干了再放油，免得吸水之后再炒干就太麻烦了。

水煮板栗我真的就是纯清水煮纯板栗，什么都不放，无糖无油，板栗本身的甜度就够了。但是，我会先用尖利的剪刀把每个板栗都剪一条口，这样煮完好剥一些。

炸薯条我嫌太慢了，把水分大致炸干应该会花很久吧，所以我从来没炸过；如果我要炸的话，可能会稍微撒点淀粉在薯条上，裹一裹粉再炸，这样表皮可能会更容易炸脆（里面还是糊状）。放了大把花椒的土豆我一想就流口水，我最爱的味道就是麻。

炒豆皮我没看明白是什么操作，为啥会那样干炒。起码要放油和辣椒面吧。我还会放蒜、孜然粉、糖、酱油、鸡精、芝麻。

烙葱花饼的时候锅里要稍微放点油啊，不然面皮直接接触铁锅都直接糊了。不过这个分层效果已经可以打 95 分了。

饼里放糖当然也是可以的，喜欢吃糖饼的话，就拿糖拌一些淀粉作为馅，包起来后擀开，包的时候一定要封好口，最好醒一会儿再擀，不然容易漏。只需要擀一次，也就是不分多层。糖饼烙出来之后里面的糖都会熔化，容易烫嘴。

糯米很顶饥，那么大的团子，一碗吃下去是不是到半夜都会撑得睡不着？

**Reply by earfanfan** [reply]
原来花生米是看颜色深浅来判定熟没熟啊。第二波一斤葛花生我已经快吃完了，当然今晚照你的建议把剩下的回锅炒了会，然而还是放凉了以后还是不脆。下回我试试先炒再放油吧。

我煮板栗也是会剪一下板栗屁股尖的。话说我妈煮板栗应该是不剪的，因为上次我特意给她煮的板栗她吃的时候还夸我煮的好，连板栗都煮得爆开了口子，哈哈哈。

我好像没有炸薯条啊。我放的那些花椒和你想象的概念不一样，拿着一个手掌大小且全开口的袋子抖了一下直接倒下去半袋，分量应该远超一大把了。

干炒豆皮我也没看明白撒情况，总之就是这事人生不会尝试第二次的。

烙葱花饼没放油是因为下锅的直接就是从冰箱拿出来的冻饼，若是放点油的话，可能会出现滋啦滋啦噼里啪啦的效果。

看到分层二字我首先想到的是教育分层、分层抽样，除此以外脑子里并没有饼也要分层的概念。擀葱花饼的方法是，先把一个团子擀成一个长方形片片状，刷油酥撒葱花，然后像叠被子一样从上下两边往中间折，折好成一条后从长条的一边开始卷，卷好以后立起来再擀成饼状。

那我做第二波饼的时候搞几个放糖的试试，其实要是如果放其他馅料是不是就成了馅饼了呢？

糯米扛饿这个到底是禾几大神的肚子吃糯米扛饿，还是糯米本身不易消化吸收？之前我的脑子里没有这个常识，就算晚上睡不着，大约也不会联想到是吃糯米团子们吃撑的。不过写到这里的时候，我突然回忆起那天那种饱腹感了……

话又说回来，今年统计之都办的R会已经，稍微有点“面目全非”的意思了（ps非贬义），对我这样的菜鸟来说不大亲民，等我再写几篇翻车记应该会有点小成，哈哈，到时候我去坛子里办个厨艺擂台赛，哈哈，让你做湖北赛区的守擂擂主。
